iteration/iteration.go

```diff
@@ -71,6 +71,17 @@ func (m Iteration) GetLastModified() time.Time {
 
 }
 
+// IsRoot Checks if given iteration is a root iteration or not
+func (m Iteration) IsRoot(spaceID uuid.UUID) bool {
+	return (m.SpaceID == spaceID && m.Path.String() == path.SepInService)
+}
+
+// Parent returns UUID of parent iteration or uuid.Nil
+// handle root itearion case, leaf node case, intermediate case
+func (m Iteration) Parent() uuid.UUID {
```

Comment:
Please add tests for this function.

Reply:
added tests in https://github.com/fabric8-services/fabric8-wit/pull/1683/commits/b7196ecbe1a12f49ad4875cb96edc0495b9aad18